<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventDialog</class>
 <widget class="QDialog" name="EventDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>164</width>
     <height>18</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>20</y>
     <width>801</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="cbPlayer">
      <property name="maxVisibleItems">
       <number>11</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbAction">
      <item>
       <property name="text">
        <string>Забил гол</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/images/images/ball.png</normaloff>:/images/images/ball.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Получил желтую карточку</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/images/images/whistle.png</normaloff>:/images/images/whistle.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Получил красную карточку</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/images/images/cards.png</normaloff>:/images/images/cards.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Получил травму</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/images/images/boots.png</normaloff>:/images/images/boots.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Заменен на</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/images/images/change.png</normaloff>:/images/images/change.png</iconset>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbChange">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EventDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EventDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
